cg/cli/get.py

```diff
@@ -139,8 +139,8 @@ def case(
         if not customer:
             LOG.error(f"{customer_id}: customer not found")
             raise click.Abort
-        status_db_cases: Iterable[Family] = status_db.families(
-            customers=[customer], enquiry=case_ids[-1]
```

Comment:
This does not make sense... If the name parameter is provided, we should not be using case_ids[-1] - right?

Reply:
it takes the last of case_ids (internal_id I am guessing). But yes if there is an enquiry given that should be used. what is upstream of this function?

Reply:
> it takes the last of case_ids (internal_id I am guessing). But yes if there is an enquiry given that should be used. what is upstream of this function?

It is a CLI function which lists details about cases :)

Reply:
> Yeah, look random as it will just pick the latest match. Seems to unspecific to be useful. What is the intended behavior?

I'm not sure what the intended behaviour is, but it would make sense if it was to search for a case based on its name if the --name parameter is provided to the CLI.

This is part of the original CLI function
```python
@get.command()
@click.option("-c", "--customer-id", help="internal id for customer to filter by")
@click.option("-n", "--name", is_flag=True, help="search family by name")
@click.option("--samples/--no-samples", default=True, help="display related samples")
@click.option("--relate/--no-relate", default=True, help="display relations to samples")
@click.option("--analyses/--no-analyses", default=True, help="display related analyses")
@click.argument("case-ids", nargs=-1)
@click.pass_context
def case(
    context: click.Context,
    customer_id: str,
    name: bool,
    samples: bool,
    relate: bool,
    analyses: bool,
    case_ids: List[str],
):
    """Get information about a case."""
    status_db: Store = context.obj.status_db
    status_db_cases: List[Family] = []
    if name:
        customer: Customer = status_db.get_customer_by_customer_id(customer_id=customer_id)
        if not customer:
            LOG.error(f"{customer_id}: customer not found")
            raise click.Abort
        status_db_cases: Iterable[Family] = status_db.families(
            customers=[customer], enquiry=case_ids[-1]
```

Reply:
Wait, name is a bool. What. Ok, I'm not changing that.